docs/app/docs/ide_integration/eclipse.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Eclipse integration
+---
+
+
+## Java
+Eclipse IDE is mostly used for development of Java projects. Therefore, in this guide we cover setting up a Java environment that's created inside devbox shell to work with Eclipse IDE.
+
+### Setting up Devbox shell
+To create a devbox shell make sure to have devbox installed. If you don't have devbox installed follow the installation guide first. Then follow the steps below:
+
+1. `devbox init` if you don't have a devbox.json in the root directory of your project.
+2. `devbox add jdk` to make sure jdk gets installed in your devbox shell.
+3. `devbox shell` to activate and go into your devbox shell.
+4. `echo $JAVA_HOME` and copy the path to your java home inside the devbox shell. It should be something like this:
+    ```bash
+    /nix/store/qaf9fysymdoj19qtyg7209s83lajz65b-zulu17.34.19-ca-jdk-17.0.3
+    ```
```

Comment:
We can now combine these into:

```shell
$ devbox shell -- 'echo $JAVA_HOME'
Installing nix packages. This may take a while...
Starting a devbox shell...
/nix/store/qaf9wyzymdij19qxyw7208s72lajz65b-zulu17.34.19-ca-jdk-17.0.3
```